opencv/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: opencv
+  version: 2.4.11
+
+source:
+  fn: opencv-2.4.11.zip
+  url: http://sourceforge.net/projects/opencvlibrary/files/opencv-unix/2.4.11/opencv-2.4.11.zip/download
+  sha1: d6e3048416d42213c204f89b9dfe39742f9a708c
+
+  patches:
+   - osx_rpath.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [not py27]
+  features:
+   - vc9  # [win and py27]
+
+requirements:
+  build:
+   - cmake
+   - numpy
+   - python 2.x
+   - eigen 3.*
+   - tbb 4.*  # [osx]
+
+  run:
+   - python 2.x
+   - numpy
```

Comment:
Also, add `x.x` here too. This makes sure we use the version of NumPy we linked against. Don't worry we use a couple.